lib/rails_critical_css_server/rewrite.rb

```diff
@@ -48,13 +48,15 @@ def original_html
       @original_html = template.capture { block.call }
     end
 
+    def extracted_css_files
+      @extracted_css_files = ExtractCssFromHtml.call(original_html)
```

Comment:
Did you mean for this to be `||=`?

I'm on vacation til tomorrow, will try to get this merged when I come back.